src/genie/libs/parser/ios/asr901/tests/ShowEnvironment/cli/equal/golden_output1_expected.py

```diff
@@ -0,0 +1,72 @@
+expected_output = {
+    "Fan": {
+        "fan1": {
+            "speed": "53",
```

Comment:
would suggest changing to `running_percent_speed` and value can be integer.
```suggestion
            "speed": 53,
```